changelog.txt

```diff
@@ -1,3 +1,27 @@
+1.9.0-beta3 (2019-02-24)
+------------------------------------------------------------------------
+- Feature: Option to adjust font size separately from GUI size (#7003)
+- Feature: Increase maximum number of orders from 64000 to ~16.7m (#7220)
+- Add: Show performance of AI and GS in framerate window
+- Add: News menu entry and shortcut for deleting all messages (#7240)
+- Change: [OSX] Improved scrolling behaviour when using touchpads
+- Change: Add scrollbar to cargo legend in cargo payment rates window
+- Change: Owner of vehicle with exclusive transport rights may now load cargo from neutral stations (#7256)
+- Change: Improved UI behaviour when dragging sliders with the mouse (#7227)
```

Comment:
was there any actual changed behaviour, or was it just fixing some fallout from ...i can't remember what, some other PR?

Reply:
Dragging doesn't completely stop if the mouse leaves the slider, so behaviour change.

Reply:
Volume sliders, there are no others.